cmd/keytransparency-server/main.go

```diff
@@ -121,11 +123,17 @@ func main() {
 	// Create gRPC server.
 	queue := mutator.MutationQueue(mutations)
 	ksvr := keyserver.New(tlog, tmap, logAdmin, mapAdmin,
-		entry.New(), auth, authz, domains, queue, mutations)
+		entry.New(), authz, domains, queue, mutations)
 	grpcServer := grpc.NewServer(
 		grpc.Creds(creds),
+		// All streaming methods are unauthenticated for now.
```

Comment:
That looks like a TODO to me.

Reply:
I don't anticipate adding authenticated streaming methods right now. 
This comment is a reminder to add the interceptor if we do add an authenticated streaming rpc.

Reply:
OK, as you're the one who'll have to write the postmortem when that doesn't happen.

Reply:
I'll add a reference to the streaming interceptor with an empty map to make things clearer.